### DISCUSSION TITLE
Unkown values on YRD226 / YRC226 / YRC246 / YRD256 / YRC256 / YRD446 by Yale

### DISCUSSION THREAD

**Discussion by asucrews** [zwave-js/zwave-js#3123] [Q&A] [answered]
For Yale YRD226 / YRC226 / YRC246 / YRD256 / YRC256 / YRD446 locks, the below parameters are undefined. I was wondering what is needed to get these parameters defined?  

[2-113-0-alarmType] Alarm Type
undefined

[2-113-0-alarmLevel] Alarm Level
undefined

[2-134-0-libraryType] Library type
undefined

[2-134-0-protocolVersion] Z-Wave protocol version
undefined

[2-134-0-firmwareVersions] Z-Wave chip firmware versions
undefined

[2-134-0-hardwareVersion] Z-Wave chip hardware version
undefined

**Comment by asucrews** [comment, marked as answer]
Hello, this is now resolved with an updated Home Assistant to 2021.08.0.  Sorry if I posted this to the wrong GitHub.

**Reply by AlCalzone** [reply]
`undefined` means that the value was either not returned (yet) or there is no value for it.

**Reply by imtifade** [reply]
I am still getting this issue. I want to use alarm levels but can't find a way to retrieve them. Any help would be great thank you. Sorry for grave digging this post.

**Reply by AlCalzone** [reply]
If you're using HA, you might have to enable the corresponding entities first. I think they are disabled by default.
